koku/masu/processor/aws/aws_cost_model_cost_updater.py

```diff
@@ -21,9 +21,9 @@
 
 from masu.database.aws_report_db_accessor import AWSReportDBAccessor
 from masu.database.cost_model_db_accessor import CostModelDBAccessor
-from masu.database.reporting_common_db_accessor import ReportingCommonDBAccessor
 from masu.external.date_accessor import DateAccessor
 from masu.util.aws.common import get_bills_from_provider
+from reporting_common import REPORT_COLUMN_MAP
```

Comment:
Do we actually need to import the `REPORT_COLUMN_MAP` for this class? I know that before we were passing in `self._column_map` to the DBAccessors, but now that they are using the constant, I can't see where this class is using the `self._column_map`.

Reply:
I removed it now. @abaiken

Reply:
@nbon12 can you look through the other classes and make sure that they don't have the same problem?

Reply:
@abaiken Yep, I found 79 other places, and I think I removed them all, let me know if you still see any.

Reply:
@abaiken actually I'm still fixing up some things.

Reply:
@abaiken OK, check now.